profiles/coreos/arm64/package.accept_keywords

```diff
@@ -2,6 +2,8 @@
 # Keep these in alphabetical order.
 
 =app-admin/sudo-1.8.20_p2 ~arm64
+=app-arch/bzip2-1.0.6-r8 ~arm64
+=app-arch/unrar-5.5.8 ~arm64
```

Comment:
I'd personally probably not accept it until needed; including it here makes it seem intentional and imply it might be used somewhere.